pkgs/by-name/du/duplicacy-web/package.nix

```diff
@@ -0,0 +1,42 @@
+{ stdenv, fetchurl, lib }:
+
+let
+  version = "1.8.1";
+  urls = {
+    x86_64-linux = "https://acrosync.com/duplicacy-web/duplicacy_web_linux_x64_${version}";
+    aarch64-linux = "https://acrosync.com/duplicacy-web/duplicacy_web_linux_arm64_${version}";
+    armv5tel-linux = "https://acrosync.com/duplicacy-web/duplicacy_web_linux_arm_${version}";
+  };
+  hashes = {
+    "x86_64-linux" = "sha256-XgyMlA7rN4YU6bXWP52/9K2LhEigjzgD2xQenGU6dn4=";
+    "aarch64-linux" = "sha256-M2RluQKsP1002khAXwWcrTMeBu8sHgV8d9iYRMw3Zbc=";
+    "armv5tel-linux" = "sha256-O4CHtKiRTciqKehwCNOJciD8wP40cL95n+Qg/NhVSGQ=";
+  };
+in
+
+stdenv.mkDerivation {
+  pname = "duplicacy-web";
+  version = version;
+
+  src = fetchurl {
+    url = urls.${stdenv.hostPlatform.system};
+    sha256 = hashes.${stdenv.hostPlatform.system};
+  };
+
+  phases = [ "installPhase" ];
```

Comment:
Never set `phases` directly. Use the `dontUnpack`, `dontBuild`, etc flags instead